app/javascript/gobierto_investments/webapp/mixins/mataro.conf.js

```diff
@@ -0,0 +1,206 @@
+  description: {
+    id: "descripcio-projecte"
+  },
+  phases: {
+    id: "estat"
+  },
+  location: {
+    id: "wkt",
+    center: [41.536908, 2.4418503],
+    minZoom: 13,
+    maxZoom: 16
+  },
+  availableFilters: [
+    {
+      id: "daterange",
+      startKey: "data-inici",
+      endKey: "data-final"
+    },
+    {
+      id: "estat",
+      multiple: true
+    },
+    {
+      id: "nom-servei-gestor",
+      multiple: true
+    },
+    {
+      id: "tipus-projecte"
+    },
+    {
+      id: "import"
+    }
+  ],
+  availableGalleryFields: [
+    {
+      id: "estat",
+      multiple: true
+    },
+    {
+      id: "data-inici",
+      filter: "date"
+    },
+    {
+      id: "data-final",
+      filter: "date"
+    },
+    {
+      id: "import",
+      filter: "money"
+    },
+    {
+      id: "tipus"
+    },
+    {
+      id: "element"
+    },
+  ],
+  availableTableFields: [
+    {
+      id: "nom-projecte"
+    },
+    {
+      id: "estat",
+      multiple: true
+    },
+    {
+      id: "import",
+      filter: "money"
+    }
+  ],
+  availableProjectFields: [
+    {
+      id: "nom-servei-gestor",
+      multiple: true
+    },
+    {
+      id: "tipus"
+    },
+    {
+      id: "element"
+    },
+    {
+      id: "tipus-projecte",
+      multiple: true
+    },
+    {
+      id: "adreca"
+    },
+    {
+      id: "adjudicatari"
+    },
+    {
+      id: "estat",
+      multiple: true
+    },
+    {
+      type: "separator"
+    },
+    {
+      id: "data-inici",
+      filter: "date"
+    },
+    {
+      id: "data-final",
+      filter: "date"
+    },
+    {
+      id: "data-adjudicacio",
+      filter: "date"
+    },
+    {
+      id: "data-inici-redaccio",
+      filter: "date"
+    },
+    {
+      id: "data-fi-redaccio",
+      filter: "date"
+    },
+    {
+      type: "separator"
+    },
+    {
+      id: "import",
+      filter: "money"
+    },
+    {
+      id: "import-adjudicacio",
+      filter: "money"
+    },
+    {
+      id: "import-liquidacio",
+      filter: "money"
+    },
+    {
+      id: "budget",
+      filter: "money",
+      type: "highlight"
+    },
+    {
+      type: "separator"
+    },
+    {
+      id: "documents",
+      type: "icon",
+      icon: {
+        title: "nom",
+        href: "url",
+        name: "file"
+      }
+    },
+    {
+      id: "tasques",
+      type: "table",
+      table: {
+        columns: [{
+          id: "nomactuacio"
+        }, {
+          id: "data"
+        }, {
+          id: "nimport",
+          filter: "money"
+        }]
+      }
+    }
+  ],
+  itemSpecialConfiguration: {
+    fn: async data => {
+      const dataArr = Array.isArray(data) ? data : [data];
+      const filterArr = dataArr.filter(d => d.attributes.partida && d.attributes["any-partida"]);
+      const allIds = filterArr.map(d => `'${d.attributes.partida}'`).join(",");
+
+      // Avoid the query if there's no code_with_zone (partida)
+      if (allIds) {
+        const endpoint = `/api/v1/data`;
```

Comment:
This endpoint is broken now, right? New url is `/api/v1/data/data`?

Reply:
Done